src/Ev.ServiceBus.Apm/Ev.ServiceBus.Apm.csproj

```diff
@@ -1,10 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFrameworks>netstandard2.1;net6.0;net8.0</TargetFrameworks>
-        <ImplicitUsings>enable</ImplicitUsings>
+        <TargetFrameworks>net6.0;net8.0</TargetFrameworks>
```

Comment:
Is there a reason behind not supporting netstandard2.1 ?

Reply:
followed the same as 
Async api and health check extensions

https://github.com/EcovadisCode/Ev.ServiceBus/blob/96d832235170b80f85b512514435231139a6f5f2/src/Ev.ServiceBus.AsyncApi/Ev.ServiceBus.AsyncApi.csproj#L4C1-L5C1